styles/base.less

```diff
@@ -5,7 +5,7 @@
 @import "ui-variables";
 @import "colors";
 
-.atom-text-editor, :host {
+.atom-text-editor, .atom-text-editor {
```

Comment:
@jglovier I did this bases on my google-fu in trying to get things updated... This is my first time editing an Atom theme so I have no idea what I am doing :P 

Although during my dev/testing it seemed to work okay.